CHANGELOG.md

```diff
@@ -25,6 +25,21 @@ Table of Contents
 
 ### Breaking Changes
 
+### Bug Fixes
+
+### New Features
+
+### Improvements
+
+
+--------------------------------------------------------------------------------
```

Comment:
Please also remove the separator here (can't comment this line): https://github.com/retest/recheck/pull/580/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR69